php/registeruser.php

```diff
@@ -0,0 +1,31 @@
+<?php
+session_start();
+include('conn.php');
+$name=$_POST['name'];
+$enroll=$_POST['enroll'];
+$username=$_POST['username'];
+$password=$_POST['password'];
+$mobile=$_POST['mobile'];
+$bhawan=$_POST['bhawan'];
+$room=$_POST['room'];
+$email=$_POST['email'];
+
+$sql="insert into student(name,enroll,username,password,mobile,bhawan,room,email)
```

Comment:
Do not directly store the password in database , encrypt the password then store its hash.
Also while comparing passwords , compare their hashes